src/app/public/modules/repeater/repeater.component.spec.ts

```diff
@@ -1311,59 +1361,37 @@ describe('Repeater item component', () => {
     it('should move an item up via keyboard controls using "Space" to activate', fakeAsync(() => {
       let items = el.querySelectorAll('sky-repeater-item');
       const itemToTest = items[1];
-      const itemDragHandle = el.querySelectorAll('.sky-repeater-item-grab-handle')[1];
-      SkyAppTestUtility.fireDomEvent(itemDragHandle, 'keydown', { keyboardEventInit: { key: ' ' } });
-      fixture.detectChanges();
-      SkyAppTestUtility.fireDomEvent(itemDragHandle, 'keydown', { keyboardEventInit: { key: 'arrowUp' } });
-      fixture.detectChanges();
-      SkyAppTestUtility.fireDomEvent(itemDragHandle, 'keydown', { keyboardEventInit: { key: ' ' } });
-      fixture.detectChanges();
```

Comment:
I encapsulated a lot of the keystrokes into a helper function `reorderItemWithKey()`. The functionality should NOT have changed.